merlin_models/tf/utils/tf_utils.py

```diff
@@ -22,9 +22,9 @@
 
 def get_output_sizes_from_schema(schema, batch_size=0, max_sequence_length=None):
     sizes = {}
-    for feature in schema.feature:
+    for feature in schema:
         name = feature.name
-        if feature.HasField("value_count"):
+        if feature._is_list:
```

Comment:
Shouldn’t this be a public property?

Reply:
Yeah - I think it probably should be .  I just left as a protected property so I wouldn't have to make changes to NVT as well. I'll update merlin-core and make the `_is_list` version deprecated

Reply:
Yeah, we've been thinking about changing this on the NVT side too. `_is_ragged` should probably also become public.

Reply:
done